pipelines/scheduled/platforms/build_windows.no_gpl.arches

```diff
@@ -1,5 +1,6 @@
-# OS       TRIPLET                    ARCH       DOCKER_ARCH    MAKE_FLAGS    TIMEOUT     DOCKER_TAG
-windows    x86_64-w64-mingw32nogpl    x86_64     x86_64         VERBOSE=1     .           v5.44
+# OS       TRIPLET                    ARCH       DOCKER_ARCH    MAKE_FLAGS      TIMEOUT     DOCKER_TAG
+windows    x86_64-w64-mingw32nogpl    x86_64     x86_64         USE_GPL_LIBS=0  .           v5.44
```

Comment:
Should we keep the `VERBOSE=1` flag?

Reply:
Let's add it, which may also help with debugging later on

Reply:
We've already add it:
https://github.com/JuliaCI/julia-buildkite/blob/d67b0dde2f69cbe5d87591dc3219594b06fe2269/utilities/build_julia.sh#L39-L40

```
Make Options:
 -> --output-sync
 -> -j8
 -> VERBOSE=1
 -> TAGGED_RELEASE_BANNER=Official https://julialang.org release
 -> JULIA_CPU_TARGET=generic;sandybridge,-xsaveopt,clone_all;haswell,-rdrnd,base(1);x86-64-v4,-rdrnd,base(1)
 -> USE_GPL_LIBS=0
```
https://buildkite.com/julialang/julia-buildkite-scheduled/builds/1149#0193c428-4844-4fae-ac5d-9c2b120ea7bc/837-841